CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+  * 0.7.0:
+    - Get latest CI status of a commit filtered by branch #96.
```

Comment:
I'd compress it a bit and put `--batch` first (with an explanation), because it's *the* highlight of the release. The idea being making it more useful for users to scan to see what's relevant to them without having to read through the referenced MRs.

``
- Feature: add `--batch` to better support  repos with many daily MRs (#116, #116)
- Fix: fix fuse() call when using experimental --use-merge-strategy to update source branch #102
- Fix: Get latest CI status of a commit filtered by branch #96 (thanks to benjamb)
- Enhancement: Check MR is mergeable before accepting MR #117 

```
Reason:

Reply:
Sure, makes sense to me.